src/pages/contact-us/SendGridForm/index.tsx

```diff
@@ -0,0 +1,217 @@
+
+        <label htmlFor="organization">
+          Organization (company or school)
+          {/* <span>*</span> */}
+        </label>
+        <input
+          type="text"
+          value={organization.val}
+          onChange={(e) => {
+            setOrganization({ ...organization, val: e.target.value });
+          }}
+          name="organization"
+          placeholder="Please enter your compony or school. (For instance, Heighliner, Inc.)"
+        />
+        <p className={styles.errorMsg}>{organization.errorMsg}</p>
+
+        <label htmlFor="role">
+          Your role
+          {/* <span>*</span> */}
+        </label>
+        <input
+          type="text"
+          value={role.val}
+          onChange={(e) => {
+            setRole({ ...role, val: e.target.value });
+          }}
+          name="role"
+          placeholder="Please enter your role. (For instance, Go developer)"
+        />
+        <p className={styles.errorMsg}>{role.errorMsg}</p>
+
+        <label htmlFor="message">
+          Message
+          {/* <span>*</span> */}
+        </label>
+        <textarea
+          rows={4}
+          name="message"
+          value={message.val}
+          onChange={(e) => {
+            setMessage({ ...message, val: e.target.value });
+          }}
+          placeholder="Please enter your message."
+        ></textarea>
+        <p className={styles.errorMsg}>{message.errorMsg}</p>
+
+        <button type="submit" className={styles.send}>
+          send
+        </button>
+      </form>
+      <div className={styles.contactUs}>
+        <Title />
+        <MediaMatrix />
+      </div>
+    </div>
+  );
+}
+
+function fieldValidator({
+  fullname,
+  organization,
+  role,
+  email,
+  message,
+}: Fields): Fields {
+  // Reset the errorMsg to empty in each field
+  const checkedFields: Fields = {
+    fullname: { ...fullname, errorMsg: '' },
+    organization: { ...organization, errorMsg: '' },
+    role: { ...role, errorMsg: '' },
+    email: { ...email, errorMsg: '' },
+    message: { ...message, errorMsg: '' },
+  };
+
+  if (fullname.val.length <= 0) {
+    checkedFields.fullname.errorMsg = "You full name can't be empty.";
+  }
+
+  if (organization.val.length <= 0) {
+    checkedFields.organization.errorMsg = "You organization can't be empty.";
+  }
+
+  if (role.val.length <= 0) {
+    checkedFields.role.errorMsg = "You role can't be empty.";
+  }
+
+  if (message.val.length <= 0) {
+    checkedFields.message.errorMsg = "The message can't be empty.";
+  }
+
+  const emailRule =
+    /^(([^<>()[\]\\.,;:\s@"]+(\.[^<>()[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
+  if (email.val.length <= 0) {
+    checkedFields.email.errorMsg = "You email can't be empty.";
+  } else if (!email.val.toLowerCase().match(emailRule)) {
+    checkedFields.email.errorMsg = "You enter email did't meet the require.";
```

Comment:
"You enter email did't meet the require."  -->   "The email you input is invalid."